practices/mosaic-css.md

```diff
@@ -57,7 +37,25 @@ h3 {
 }
 ```
 
-There is a lot of crossover between OOCSS and Atomic CSS. We create atomic classes to handle positioning (default position/float, standardizing margins) and common style elements such as colors, backgrounds and borders, such as in the example below:
+We make use of a collection of single purpose styling classes (single responsibility for maximum reuse) that fits well with componentized templates. These classes and their associated styling are immutable, meaning you'd use the same classes across projects. In other words, The aim is to create a common "vocabulary" meant to style documents regardless of context or content.
+
+An example of how to use atomic classes would be with font sizes. We could write the following CSS classes that can then be used across any project to set your font-sizes:
+
+```css
+.text11 {
+    font-size: 1.1rem;
+}
+
+.text13 {
+    font-size: 1.3rem;
+}
+
+.text14 {
+    font-size: 1.4rem;
+}
+```
+
+We create atomic classes to handle positioning (default position/float, standardizing margins) and common style elements such as colors, backgrounds and borders, such as in the example below:
```

Comment:
Shouldn't we be saying "utility classes" rather than "atomic classes" to get away from the confusion we've had with Atomic CSS